server/core/src/configuration_hook_manager.cpp

```diff
@@ -0,0 +1,22 @@
+#include "irods/configuration_hook_manager.hpp"
+#include <fmt/format.h>
+#include "irods/irods_logger.hpp"
+namespace irods::experimental::hooks{
+  configuration_hook_manager::configuration_hook_manager(){}
+  configuration_hook_manager &configuration_hook_manager::instance(){
+    static configuration_hook_manager i;
+    return i;
+  }
+  void configuration_hook_manager::add_hook(hook a){
+    hooks_.push_back(a);
+  }
+  void configuration_hook_manager::run_hooks(const std::vector<nlohmann::json>& changes){
+    irods::experimental::log::server::info(fmt::format("Found {} changes", changes.size()));
```

Comment:
`fmt::format` isn't necessary.

The logger supports the same interface as fmtlib (mostly).